src/modules/transaction/api/index.test.js

```diff
@@ -11,10 +11,8 @@ import accounts from '@tests/constants/wallets';
 import { genKey, blsKey, pop } from '@tests/constants/keys';
 import { mockAppTokens } from '@tests/fixtures/token';
 import {
-  getTransaction,
```

Comment:
@soroushm `getTransaction` was removed from the test but the util in the index file wasn't removed